src/arith_uint256.cpp

```diff
@@ -173,8 +173,8 @@ unsigned int base_uint<BITS>::bits() const
 {
     for (int pos = WIDTH - 1; pos >= 0; pos--) {
         if (pn[pos]) {
-            for (int bits = 31; bits > 0; bits--) {
-                if (pn[pos] & 1 << bits)
+            for (size_t bits = 31; bits > 0; bits--) {
+                if (pn[pos] & 1U << bits)
```

Comment:
Please write this as `pn[pos] & (1U << bits)`.

Reply:
Fixed.